test/run_test

```diff
@@ -658,16 +660,19 @@ run_migration_test ()
     [ "${OS}" == "fedora" ] && return 0
 
     local from_version
-    if [ "${OS}" == "rhel8" ]; then
-        local upgrade_path="9.6 10"
-    else
+    # Only test a subset of the migration path on non-intel hosts
+    if [ "$(uname -i)" == "x86_64" ]; then
         local upgrade_path="9.2 9.4 9.5 9.6 10"
+    else
+        local upgrade_path="10"
```

Comment:
This means the latest version and therefore I would prefer to use
```
local upgrade_path="$VERSION"
```

so it will not be updated during each new version.

Reply:
This will be upgraded for each new version since the upgrade path will grow as we add new images. Check #343 for example.